filebeat/_meta/config/filebeat.inputs.reference.yml.tmpl

```
@@ -568,8 +568,18 @@ filebeat.inputs:
   # You may wish to have separate inputs for each service. You can use
   # include_matches to specify a list of filter expressions that are
   # applied as a logical OR. You may specify filter
-  #include_matches:
-    #- _SYSTEMD_UNIT=foo.service
+  #include_matches.or:
+    #- equals:
+      #- _SYSTEMD_UNIT=foo.service
```

I believe we can make it simpler by using `include_matches.equals` directly:
```suggestion
  #include_matches.equals:
    #- _SYSTEMD_UNIT=foo.service
```

What do you think?

---

It is indeed simpler, but I think the way I wrote it is clearer. I choose clarity over simplicity in the documentation. :)